extension/lib/magento/customer/getItems.js

```diff
@@ -13,8 +13,13 @@ module.exports = async (context, input) => {
   const wishlistItems = await request.send(wishlistItemsEndpointUrl, 'Request to Magento: getFavorites', 'GET')
   context.storage.user.map.del('wishlistItemIdMapping')
   const productIds = wishlistItems.map(item => {
-    context.storage.user.map.setItem('wishlistItemIdMapping', item.product_id, item.wishlist_item_id)
-    return item.product_id
+    const { child_ids: childIds, type, product_id: productItemId } = item
```

Comment:
Maybe just use this and delete the line below?
```suggestion
    let { child_ids: childIds, type, product_id: productId } = item
```